CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 49.7.0
```

Comment:
I would leave the version heading out at the moment as, although unlikely, another PR may get merged before making the release which changes this to a major update. The release PR can add the heading.